### PR TITLE
docs: add Darklang to showcase

### PR DESCRIPTION
**Pull request by pbiggar**

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -383,6 +383,15 @@ const Users: User[] = [
...
     tags: ['opensource', 'i18n', 'large'],
   },
+  {
+    title: 'Darklang',
+    description:
+      'A new way of building serverless backends, with no infra, framework or deployment nightmares.',
+    preview: require('./showcase/darklang.png'),
+    website: 'https://docs.darklang.com/',
+    source: 'https://github.com/darklang/docs',
+    tags: ['product'],
```
Comment (slorber) [inline website/src/data/users.tsx:393]:
```suggestion
    tags: ['product', 'opensource'],
```

**Comment by slorber** [comment]
Thanks, LGTM

Can you please sign the CLA? Required by Meta/FB

**Comment by pbiggar** [comment]
> Can you please sign the CLA? Required by Meta/FB

Done, thanks!

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**